presto-docs/src/main/sphinx/include/PrestoThriftService.thrift

```diff
@@ -0,0 +1,305 @@
+namespace java.swift com.facebook.presto.connector.thrift.api
```

Comment:
I don't think so. I can override it if necessary. Basically, it says that swift classes are in this namespace. It doesn't mean that classes for other languages should be in the same package.